injected/integration-test/test-pages/favicon/config/favicon-absent.json

```diff
@@ -1,6 +1,9 @@
 {
   "features": {
-
+    "themeColor": {
+      "state": "disabled",
+      "exceptions": []
```

Comment:
Is the exceptions property being used or planned for in the future? If not, I think we can get rid of it.

Reply:
Scratch that. Just saw that it's a standard property of top-level features in privacy config.